lib/server/server.js

```diff
@@ -0,0 +1,37 @@
+/*!
+ * webpack-jarvis <https://github.com/zouhir/webpack-jarvis>
+ *
+ * Copyright (c) 2017, Zouhir C.
+ * Licensed under the MIT License.
+ */
+
+const polka = require("polka");
+const socket = require("socket.io");
+const statics = require("serve-static");
+const { join } = require("path");
+
+const client = join(__dirname, "..");
+
+exports.init = (compiler, isDev) => {
+  let app = polka().get("*", (req, res) => {
+    // TODO: fix Polka
+    statics(client)(req, res, err => {
+      res.statusCode = 404;
+      res.end('');
+    });
+  });
+
+  if (isDev) {
+    compiler.outputPath = "/"; // wtf?
```

Comment:
Why wtf?

Reply:
The `webpack-dev-middleware` is relying on a computed `compiler.outputPath` instead of `compiler.output.path`, which is what's actually passed thru via config.

The problem is the `compiler.outputPath` is computed _before_ the config is read, it seems.

Reply:
Weird. Sounds like you have a PR for webpack. It'd be cool to have a more descriptive comment.

Reply:
Thought about the PR, but efforts are focused on `v4` right now. If this persists after that stabilizes, I'll poke them about it 👍

Reply:
Would love to see that